app/test_main.py

```diff
@@ -11,10 +11,10 @@
     ("https://www.google.com", False, True, "Not accessible")
 ])
 def test_valid_url_and_connection_exists(monkeypatch: MonkeyPatch,
```

Comment:
Perhaps this indentation style will provide better readability for function with long names:
```
def test_valid_url_and_connection_exists(
    monkeypatch: MonkeyPatch,
    url: str,
    expected: str,
    valid: bool,
    connection: bool
) -> None:
    pass
```

Reply:
Agree, on the other hand you should keep the same bracket open/close style f.e.:
(something,
something)
or
(
something,
something
)